src/service/SubscriptionService.java

```diff
@@ -45,10 +45,9 @@ public void addSubscription(int s_id, boolean status, Date from,
     public void addSubscribesTo(int mbid, int s_id) throws SQLException {
         PreparedStatement ps;
         try {
-            ps = con.prepareStatement("INSERT INTO Subscribes_To VALUES " +
-                    "(?, ?)");
-            ps.setInt(1, mbid);
-            ps.setInt(2, s_id);
+            ps = con.prepareStatement("INSERT INTO Subscribes_To VALUES (?, ?)");
+            ps.setInt(1, s_id);
```

Comment:
Should the parameters be switched so it's ```addSubscribesTo(int s_id, int mbid)``` since we're now setting the s_id first. Or does it not matter?

Reply:
it doesn't matter as long as we make sure the first parameter in the values is an s_id and the second is an mbid - it would be nicer to make it consistent but I didn't want to mess up any existing calls you guys might've add by switching them around